flink-tests/src/test/java/org/apache/flink/test/operators/ExecutionEnvironmentITCase.java

```diff
@@ -66,6 +74,49 @@ public void mapPartition(Iterable<Integer> values, Collector<Integer> out) throw
 		assertEquals(PARALLELISM, resultCollection.size());
 	}
 
+	@Test
+	public void testAccessingBlockingPersistentResultPartition() throws Exception {
```

Comment:
I am not sure if this test is well placed in the ExecutionEnvironmentITCase.
It does not test the ExecutionEnvironment, it tests a different feature.

Reply:
I'll move this test to a new ITCase and try to add some unit tests.